geps/gep-1494/index.md

```diff
@@ -130,20 +130,383 @@ From @ongy, some additional goals to keep in mind:
 
 ## API
 
-(... details, can point to PR with changes)
+This GEP proposes a two-part solution to this problem:
+
+* We introduce a new HTTPRoute Filter, `ExternalAuth`, that allows the
+  specification of an external source to connect to using Envoy's `ext_auth` protocol.
+* We introduce a new Policy object that can be targeted at either the
+  Gateway or HTTPRoute levels. In either of these cases, it _defaults_ the settings
+  for the HTTPRoute Filter across all HTTPRoute matches that roll up to the object.
+
+These two parts will be done in two separate changes - Filter first, then
+Policy after.
+
+Both of these additions use the same underlying struct for the config, so that
+changes or additions in one place add them in the other as well.
+
+This plan has some big things that need explaining before we get to the API details:
+
+* Why a Filter plus Policy approach?
+* Why two changes?
+* Why Envoy's `ext_auth`?
+
+### Why a Filter plus Policy approach?
+
+We have two main requirements: Ana needs to be able to configure auth at least at
+the smallest possible scope, and Ana, Ian and Chihiro need to be able to configure
+defaults at larger scopes.
+
+The smallest possible scope for this config is the HTTPRoute Rule level, where
+you can match a single set of matchers - like a path, or a path and header
+combination.
+
+At this level, the inline tool we have available to perform changes is the HTTPRoute
+Filter, which also has the property that it's designed to _filter_ requests. This
+matches the overall pattern here, which is to _filter_ some requests, allowing
+or denying them based on the presence of Authentication and the passing of
```

Comment:
I'm still a bit confused on the proper noun Authentication here. Is Authentication something that just ext_authz is supposed to do (in conjunction with its authorization role), or is Authentication also something that happens elsewhere in Gateway, for example via frontend mTLS? 

I know we talked about this as Kubecon but I dont' recall the conclusion.